### ISSUE TITLE
Release 5.0.0-beta.1

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1536]
- [x] Address all coding and style violations detected during a *Release* build (send a regular pull request which must be merged before continuing)
- [x] ensure completeness of draft [release notes](https://github.com/FakeItEasy/FakeItEasy/releases), including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
- [x] run `tools\prepare_release.cmd x.y.z` to:
    - change "vNext" names to appropriate x.y.z version for the next GitHub Release, milestone, and release checklist (this issue)
    - create a new draft GitHub Release, milestone, and release checklist
- [x] tag master with the new version number and push to upstream. This will cause the build server to
    - push the nupkgs to [nuget.org](https://www.nuget.org/profiles/FakeItEasy) and
    - attach all nupkgs from the tagged build to the GitHub Release.
- [x] wait for NuGet package validation
- [x] de-list pre-release NuGet packages if present
- [x] publish the GitHub release
- [x] tweet, mentioning contributors, and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] if this is **not** a pre-release, post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [x] close this issue
- [x] if this is **not** a pre-release, close the [milestone](https://github.com/FakeItEasy/FakeItEasy/milestones?state=open) for this release

**Comment by thomaslevesque** [comment]
Additional item for this release:
[ ] Manually create and publish a dummy version of the analyzer metapackage as described in #1509

**Comment by blairconrad** [comment]
I'm inclined to release in order to make the content of the 4.9.2 hotfix available for our early adopters as well. Any reason not to tomorrow? Should we go alpha.2 or beta.1?

**Comment by blairconrad** [comment]
> Manually create and publish a dummy version of the analyzer metapackage

Do you want this done for this release, or do you mean to do by the time we release 5.0.0? If the latter, no worries. I'll copy forward.

**Comment by blairconrad** [comment]
Oh, sneaky!
Do you want to do the honours sometime, since you know what you're doing? I don't think timing matters.

**Comment by blairconrad** [comment]
Late-breaking news. I'd like to fix #1548 before we release.

**Comment by blairconrad** [comment]
If that gets in, I've a potential tweet

```
Get FakeItEasy 5.0.0-beta.1!
- exceptions thrown from Invokes propagate correctly (thanks to @urWaldfee for reporting)
- call configuration that includes array creation no longer fails with type mismatch

https://github.com/FakeItEasy/FakeItEasy/releases/5.0.0-beta.1
```

**Comment by thomaslevesque** [comment]
> Do you want to do the honours sometime, since you know what you're doing? I don't think timing matters.

I can probably do it today or tomorrow.

> I'd like to fix #1548 before we release.

👍

**Comment by thomaslevesque** [comment]
I added the analyzer meta-package to the [5.0.0-beta.1 release](https://github.com/FakeItEasy/FakeItEasy/releases/tag/untagged-ce7e7e1ae0bc8e4ba263), since the build won't do it for this package.
